Firestore/core/src/firebase/firestore/model/mutation_batch_result.cc

```diff
@@ -33,7 +33,7 @@ MutationBatchResult::MutationBatchResult(
     MutationBatch batch,
     model::SnapshotVersion commit_version,
     std::vector<model::MutationResult> mutation_results,
-    absl::optional<nanopb::ByteString> stream_token)
+    nanopb::ByteString stream_token)
```

Comment:
Sorry, I should have highlighted this in the change description.

Proto3 treats empty and null strings interchangeably, so there's no value in distinguishing these cases, especially because this value exists only as something we get from the server and return to the server.

Resume tokens are another, similar `ByteString` quantity and they are also non-optional (i.e. just `nanopb::ByteString`) so I changed this to match and simplify the types.